libs/platform/src/lib/thumbnail/thumbnail-image/thumbnail-image.component.ts

```diff
@@ -76,22 +70,10 @@ export class ThumbnailImageComponent implements OnChanges, OnInit {
     }
 
     /** Opens the Dialog when the imgaes croses the maximum number of images to display */
-    openDialog(selectedMedia: Media, mediaList: Media[]): void {
+    openDialog(media: Media): void {
```

Comment:
changes in this file appear to be not directly related to the migration to Angular 13. Can you please explain why are they here? Same for corresponding html of this component and `thumbnail-component` files

Reply:
From the pr description

```
Components refactoring to avoid NG3003: Import cycles would need to be created to compile this component because of compiling library in partial mode from now. Please take a look when reviewing.
```

So the only reason to resolve circular deps.